tabs/src/types/global.d.ts

```diff
@@ -28,7 +28,7 @@ interface Wallet {
 
 type WalletType = 'Sending' | 'Receiving';
 
-declare module '*.svg' {
-  const content: any;
-  export default content;
+declare module '*.module.css' {
```

Comment:
Wondering if this should go in here or not?

Reply:
resolved